Apps/WebClient/src/ClientApp/src/services/loggingService.ts

```diff
@@ -0,0 +1,42 @@
+import Winston from "winston";
+import { ILoggingService } from "@/services/interfaces";
+import { injectable } from "inversify";
+
+@injectable()
+export class LoggingService implements ILoggingService {
```

Comment:
Since this is a Winston specific implementation, how about naming it like so WinstonLoggingService

Reply:
I were wondering if LoggingService is a long name as we will use it everywhere in our code. I wanted to change it to Logger but weren't sure :-) I would like to keep this name or change to a shorter name if you don't mind.

Reply:
Well since it's not a service per se we can change it. Following the c# one how about just ILogger and name the implementation WinstonLogger?
Since this will be used in a dependency injection pattern. the name of the implementation will not show up on the files that use it.
Like
```
const logger: ILogger= container.get<ILogger>(
    SERVICE_IDENTIFIER.Logger
);
...
logger.info(...);
```

Reply:
good idea. I still want to get rid of the Winston as if we decide to change the log lib in few year, we don't have to update the WinstonLogger to OtherLogger everywhere.

Reply:
updated code per @FuriousLlama 's suggestions above